recipes/numpy/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cat > site.cfg <<'siteconfig'
+[DEFAULT]
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+
+[atlas]
+atlas_libs = openblas
+libraries = openblas
+
+[openblas]
+libraries = openblas
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+siteconfig
```

Comment:
Could you please implement your thoughts in a PR against this branch?  I really don't know what you're proposing.  Sorry...